srcpkgs/libnma/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'libnma'
+pkgname=libnma
+version=1.8.30
+revision=1
+build_style=meson
+build_helper="gir"
+hostmakedepends="cmake gettext glib-devel gtk-doc pkg-config vala"
+makedepends="gcr-devel gtk+3-devel iso-codes
+ mobile-broadband-provider-info NetworkManager-devel"
+depends="NetworkManager iso-codes"
+short_desc="NetworkManager panel applet for GNOME - runtime library"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="GPL-2.0-only"
```

Comment:
Looks like a non-trivial amount of the project is LGPL, so change this line to:

`license="GPL-2.0-only, LGPL-2.1-or-later"`

Reply:
Thank you, I've changed the license like you said.